net/coeurl/Portfile

```
@@ -24,6 +24,14 @@ meson.wrap_mode     default
 # Fix subproject wrap file for curl: [provides] should be [provide]
 patchfiles-append   patch-subproj-curl.diff
 
+# Backport of:
+# https://github.com/gabime/spdlog/commit/c65aa4e4889939c1afa82001db349cac237a13f8
+pre-build {
+    set spdlog_v    1.8.5
+    system -W ${worksrcpath}/subprojects/spdlog-${spdlog_v} \
```

You could avoid the need for specifying the version by globbing, in case they ever change the version. (Untested.)
```suggestion
    system -W {*}[glob ${worksrcpath}/subprojects/spdlog-*] \
```

---

I will try that, thank you.